tsconfig.json

```diff
@@ -17,8 +17,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitAny": true,
-    "rootDirs": ["src", "stories"]
+    "rootDirs": ["src"]
   },
-  "include": ["src"],
-  "exclude": ["build", "node_modules"]
```

Comment:
❓ Removing the exclusion was on purpose?

Reply:
Yes because as per documentation, `exclude` is only to override `include`.